docs/sources/reference/components/otelcol/otelcol.processor.k8sattributes.md

```diff
@@ -143,12 +156,30 @@ The `annotation` block configures how to extract Kubernetes annotations.
 
 {{< docs/shared lookup="reference/components/extract-field-block.md" source="alloy" version="<ALLOY_VERSION>" >}}
 
+{{< admonition type="warning" >}}
+
+The `regex` argument is deprecated and may be removed in upcoming versions.
+Use the [ExtractPatterns](extract-patterns) function from `otelcol.processor.transform` instead. 
+
+[extract-patterns]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/{{< param "OTEL_VERSION" >}}/pkg/ottl/ottlfuncs/README.md#extractpatterns
```

Comment:
@clayton-cornell This link doesn't seem to work, but I don't know why. It's almost identical to a similar link in the `otelcol.processor.transform` doc. Any ideas why this is?

The same link is also present in the next section btw.

Reply:
Fixed via @dehaansa's [suggestion](https://github.com/grafana/alloy/pull/2314#discussion_r1897082324).